pipeline/dags/mrms_download.py

```diff
@@ -0,0 +1,39 @@
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+from datetime import timedelta
+
+# default arguments for each task
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+    'catchup': False
+}
+
+with DAG(
+    'weather_MRMS_data',
+    default_args=default_args,
+    description='Download MRMS data',
+    schedule_interval=timedelta(days=1),
```

Comment:
This should run quite often (every 5-20 minutes) to minimize delay before we get the data. It should only do work if there is something that needs to be downloaded. I will be adding an interlock so that if an earlier invocation is still running, the task will abort quickly.